notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+    "ax1.set_xlim(2000, 3000)\n",
+    "ax1.set_ylim(2000, 1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, these circular apertures don't fit our data very well. After all, this is the Hubble eXtreme Deep Field, so there aren't any nice, round, nearby Milky Way stars in this image! \n",
+    "\n",
+    "Let's use ellipses instead, to better match the morphology of the galactic blobs in our image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Elliptical Apertures"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from photutils import (detect_sources, source_properties, \\\n",
+    "                       EllipticalAnnulus, EllipticalAperture)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In a [previous notebook](../02_source_detection/02_source_detection.ipynb), we showed how you can use the `photutils.detect_sources` [feature](https://photutils.readthedocs.io/en/stable/api/photutils.detect_sources.html) to generate segmentation maps, which identify and label contiguous (connected) objects within an image. Then, with `source_properties` [feature](https://photutils.readthedocs.io/en/stable/api/photutils.segmentation.source_properties.html?highlight=source_properties), you can access descriptive properties for each unique object - not just their centroid positions, but also their pixel areas, eccentricities, orientations with respect to the coordinate frame of the image, and more.\n",
+    "\n",
+    "Here we'll use the centroid, semimajor axis, semiminor axis, and orientation values from `source_properties` to generate elliptical apertures for each of the sources in our image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# Define threshold and minimum object size\n",
+    "threshold = 5. * std\n",
+    "npixels = 15\n",
+    "\n",
+    "# Create a segmentation image\n",
+    "segm = detect_sources(xdf_image.data, threshold, npixels)\n",
+    "\n",
+    "# Create a catalog using source properties\n",
+    "catalog = source_properties(xdf_image.data, segm)\n",
+    "table = catalog.to_table()\n",
+    "\n",
+    "# Display the table\n",
+    "table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r = 3.  # pixels; approximate isophotal extent of semimajor axis\n",
+    "\n",
+    "# Create the apertures\n",
+    "elliptical_apertures = []\n",
+    "for obj in catalog:\n",
+    "    position = (obj.xcentroid.value, obj.ycentroid.value)\n",
+    "    a = obj.semimajor_axis_sigma.value * r\n",
+    "    b = obj.semiminor_axis_sigma.value * r\n",
+    "    theta = obj.orientation.value\n",
```

Comment:
Not sure exactly where the issue is, but I'm assuming the angle is the issue... But in this cell and the figure below, the elliptical axes are not aligned to the ellipse of the galaxies... although one is, somehow (at x=2650, y=1750). There is a difference in how the angle is defined in `EllipticalAperture` and `source_properties` (and `SourceProperties`, the class the function calls in turn); one defines the angle as clockwise, the other counter-clockwise...